### PR TITLE
Add Database migration section to install instructions

### PR DESCRIPTION
**Pull request by HemKal**
Issue #824 Install instructions missing Migration instructions

### CODE REVIEW

**Comment by HemKal** [comment]
This PR is Replacing #826

File: INSTALL.md [hunk 1]
```diff
@@ -327,6 +328,25 @@ xa-data-source add \
   --xa-datasource-properties=ServerName=localhost,PortNumber=5432,DatabaseName=psm
 EOF
 ```
+## Run Database Migrations
+Setting up the database requires running the liquibase migrations to 
+get the database into the up-to-date configuration. See [MIGRATIONS.md](docs/MIGRATIONS.md) for details
```
Comment (PaulMorris) [inline INSTALL.md]: Just missing a period after 'for details' here.

**Comment by jasonaowen** [comment]
Thanks for addressing my concerns, @HemKal!

File: INSTALL.md [hunk 2]
```diff
@@ -327,6 +328,25 @@ xa-data-source add \
...
+
+    ```shell
+       > ./gradlew db:update
+    ```
+
+2. Add the sample data to the database, use `psm-app/db/seed.sql`:
+
+    ```ShellSession
+       $ cat {/path/to/psm}/psm-app/db/seed.sql \
+        | psql -h localhost -U psm psm
```
Comment (cecilia-donnelly) [inline INSTALL.md]: I think this also needs a reference to jbpm.sql.

Comment (HemKal) [reply INSTALL.md]: @cecilia-donnelly  any changes to jbpm.sql should be part of migration. Where seed.sql is reseeding the data on any migration changes. Correct me if I am wrong. Thx

**Comment by cecilia-donnelly** [comment]
I edited this since @HemKal is out of town for a while and pushed to [branch pr/834](https://github.com/SolutionGuidance/psm/tree/pr/834).  I probably can't incorporate those changes into this PR, though, since I don't have push rights to @HemKal's fork.  Thoughts, @PaulMorris?

**Comment by PaulMorris** [comment]
@cecilia-donnelly Your edits look good to me.  It'd be good to get these updates in sooner than later, so I'd say feel free to open a new PR (or push directly to master?) and then close this one, or however you see fit.

**Comment by cecilia-donnelly** [comment]
Removed the pr/834 branch in favor of [834-migration-updates](https://github.com/SolutionGuidance/psm/tree/834-migration-updates).

**Pull request closed without merging**